Source/DafnyCore/Resolver/NameResolutionAndTypeInference/NameResolutionAndTypeInference.cs

```diff
@@ -5894,7 +5894,8 @@ public MethodCallInformation ResolveApplySuffix(ApplySuffix e, ResolutionContext
               }
               if (allowMethodCall) {
                 Contract.Assert(!e.Bindings.WasResolved); // we expect that .Bindings has not yet been processed, so we use just .ArgumentBindings in the next line
-                var tok = Options.Get(Snippets.ShowSnippets) ? e.Origin.ToToken() : e.Tok;
+                // TODO simplify to e.Origin
```

Comment:
Same question here. You can answer in only one spot.

Reply:
Processed the TODO